tools/postinstall-android.js

```diff
@@ -123,13 +122,11 @@ shelljs.echo("allprojects {\n\trepositories {\n\t\tmavenCentral\(\)\n\t}\n}").to
 var oldBuildScriptDepTree = "buildscript {";
 var newBuildScriptDepTree = "buildscript {\n\tdependencies {\n\t\tclasspath 'com.android.tools.build:gradle:1.3.1'\n\t}\n";
 shelljs.sed('-i', oldBuildScriptDepTree, newBuildScriptDepTree, path.join(appProjectRoot, 'build.gradle'));
-var oldLibDep = "compile \"com.android.support:support-v13:23+\"";
-var newLibDep = "compile \"com.android.support:support-v13:23+\"\ncompile project(':SalesforceHybrid')";
-shelljs.sed('-i', oldLibDep, newLibDep, path.join(appProjectRoot, 'build.gradle'));
+var newLibDep = "compile \"com.google.android.gms:play-services:7.5.0\"\ncompile project(':SalesforceHybrid')";
```

Comment:
This is one big change. The ```PushPlugin``` requires ```GMS``` (probably because ```Google``` has moved it out to it's own library and deprecated ```C2DM```).